redbot/cogs/reports/reports.py

```diff
@@ -175,7 +175,10 @@ def pred(m):
         if await self.bot.embed_requested(channel, author):
             em = discord.Embed(description=report)
             em.set_author(
-                name=_("Report from {0.display_name}").format(author), icon_url=author.avatar_url
+                name=_("Report from {author}{maybe_nick}").format(
```

Comment:
A bit of a questionable formatting job here?

If I'm reading this code correctly this would make a username appear as the following:
``KowlinNickname``

Reply:
Wiith how maybe_nick is passed to that, you'll either just get:

`Username#discrimantor` if no nickname, of `Username#discriminator (Nickname)` if they have one (making it more clear on nicknames, since those are more likely to change)

Reply:
Ah right, this relied on me thinking that the author object would only return the username and not the full user-discrim.